doc/object_properties.adoc

```diff
@@ -0,0 +1,300 @@
+* default value: `{a = 255, r = 255, g = 255, b = 255}`
+
+The color that the nametag text will be.
+
+TIP: This property can be set to `false` to disable it.
+
+=== `nametag_bgcolor`
+* field type: `color_def` or `false`
+* internal type: `ColorSpec`
+
+The color that the nametag's background will be.
+
+TIP: This property can be set to `false` to disable it.
+
+=== `automatic_face_movement_max_rotation_per_sec`
+* field type: `f32`
+* default value: `-1.0`
+
+Cap the automatic rotation to this value. If the value is less than or equal to 0, there will be no maximum.
+
+=== `infotext`
+* field type: `string`
+
+When a player points at the object, they will be shown the text. The maximum number of characters is the `u16` max value.
+
+=== `static_save`
+* field type: `bool`
+* default value: `true`
+
+Whether the object is saved statically, if not, it will be deleted when the world block is unloaded.
+
+=== `wield_item`
+* internal type: `ItemString`
+
+Only when using "wielditem" visual. The maximum number of characters is the `u16` max value.
+
+=== `use_texture_alpha`
+* field type: `bool`
+* default value: `false`
+
+Whether or not the texture's alpha should be used.
+
+=== `shaded`
+* field type: `bool`
+* default value: `true`
+
+Controls whether or not diffuse lighting is applied to the object.
+
+=== `show_on_minimap`
+* field type: `bool`
+* default value: `false`
+
+Controls whether or not the object is visible on the minimap.
+
+=== `damage_texture_modifier`
+* field type: `string`
+* default value: `"^[brighten"`
+
+A texture modifier that will be appended to the object's current textures for the duration of the damage flash when the object is damaged.
+
+== Layout of Object Properties
+
+The way these properties are laid out is in a table.
+
+NOTE: Extraneous properties will be ignored, but might be interpreted incorrectly by furutre versions of the engine, which is why it is recommended to prefix them with an underscore.
+
+NOTE: This uses regular indexing so metatables work as expected.
+
+== Example
+
+.Object Properties table example
+====
+[source,lua]
+----
+{
+    hp_max = 10,
+    breath_max = 5,
+    eye_height = 0.8,
+    zoom_fov = 0,
+    physical = true,
+    collide_with_objects = true,
+    collisionbox = {-0.3, -0.3, -0.3, 0.3, 0.3, 0.3},
+    selectionbox = {-0.3, -0.3, -0.3, 0.3, 0.3, 0.3},
+    pointable = true,
+    visual = "mesh",
+    mesh = "mesh.obj",
+    visual_size = {x = 1, y = 1, z = 1},
+    textures = {"texture.png", "texture.png"},
+    colors = {},
+    spritediv = {x = 1, y = 1},
+    initial_sprite_basepos = {x = 0, y = 0},
+    is_visible = true,
+    makes_footstep_sound = true,
+    stepheight = 0.4,
+    automatic_rotate = 0.0,
+    automatic_face_movement_dir = 0,
+    backface_culling = true,
+    glow = 0,
+    nametag = "My object",
+    nametag_color = {r = 255, g = 255, b = 255},
+    nametag_bgcolor = {r = 0, g = 0, b = 0},
+    automatic_face_movement_max_rotation_per_sec = 0,
```

Comment:
Please sort this properly. This property should be next to `automatic_face_movement_dir`.